packages/core/test/forms/numericInputTests.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { assert } from "chai";
+import { mount } from "enzyme";
+import { spy } from "sinon";
+import * as React from "react";
+
+import { NumericInput } from "../../src";
+
+describe("<NumericInput>", () => {
```

Comment:
there are already tests for this component in `packages/core/test/controls/numericInputTests.tsx`. can you please move your new tests into that test suite?